### PR TITLE
docs: use media queries

### PR DESCRIPTION
**Pull request by NazarPauk**
## Description

Provided rules for using media queries and utilizing breakpoints

### CODE REVIEW

File: blocks/README.md [hunk 1]
```diff
@@ -40,3 +40,27 @@ blocks/
 ```
 
 Please refer to the official [BEM](https://en.bem.info/methodology/quick-start/#introduction) or [BEM Nested](https://en.bem.info/methodology/quick-start/#file-structure) documentation for details if needed.
+
+### Using media queries
+
+For all components, we use the Desktop-first design approach, programming them initially for the desktop version, and subsequently adapting them to breakpoints: `1199px, 659px,` using `max-width` for media queries
```
Comment (naumch1k) [inline blocks/README.md:46]: @NazarPauk Great job on updating the docs and including information about the use of media queries! 

I'd like to suggest to rephrase it a bit to improve the understanding of our approach. When mentioning the breakpoints, it's important to clarify that the values `659px` and `1199px` represent the upper limits of the screen sizes targeted by the media queries rather than being breakpoints themselves.

For example, instead of saying _"breakpoints: 1199px, 659px,"_ we could rephrase it to emphasize that these values are the upper limits of our ranges. Something like: _"Our breakpoints are set at` 659px` and `1199px`, representing the upper limits of the screen sizes targeted by our media queries."_

File: blocks/README.md [hunk 2]
```diff
@@ -40,3 +40,27 @@ blocks/
...
+
+Example:
+
+```bash
```
Comment (naumch1k) [inline blocks/README.md:50]: @NazarPauk Let's make a minor formatting adjustment to make the code example in the documentation more precise and consistent.

Please replace 'bash' with 'CSS' after the triple backticks (```) to indicate that the code block contains CSS styling.  Also, let's use `/* */` for comments within the CSS code block.

This adjustment will bring some familiar CSS styling to our code example, making it easier to read and work with.

File: blocks/README.md [hunk 3]
```diff
@@ -40,3 +40,27 @@ blocks/
 ```
 
 Please refer to the official [BEM](https://en.bem.info/methodology/quick-start/#introduction) or [BEM Nested](https://en.bem.info/methodology/quick-start/#file-structure) documentation for details if needed.
+
+### Using media queries
```
Comment (naumch1k) [inline blocks/README.md:44]: @NazarPauk What do you think about revising the document's structure to improve its readability?

We can add a second-level heading titled _"Files and Folders Structure"_ before line 3 and make _"Using media queries"_ a second-level heading too. This should make it easier to navigate and understand.

**Review by naumch1k — CHANGES_REQUESTED** [review]
@NazarPauk Great work updating the documentation! I appreciate your efforts in clarifying the usage of media queries. Please review the suggested enhancements to ensure the document is even more user-friendly and organized.

File: blocks/README.md (hunk 1 above)
Comment (NazarPauk) [reply blocks/README.md]: Done✅

File: blocks/README.md (hunk 2 above)
Comment (NazarPauk) [reply blocks/README.md]: Done✅

File: blocks/README.md (hunk 3 above)
Comment (NazarPauk) [reply blocks/README.md:44]: Do you mean to document the nesting of folders and files using the BEM Nested methodology in a document?

Comment (naumch1k) [reply blocks/README.md:44]: @NazarPauk I'm sorry, I am not sure if I got your question right.. we got two parts to this README: BEM Nested and media requests. I propose to add second level heading _Files and Folders Structure_ on line 3. And also to make _Using media queries_ second-level heading.

Comment (naumch1k) [reply blocks/README.md:44]: @NazarPauk, please let me know if you want to hop on a quick call so I can explain it better. It's really just a tiny 5-second adjustment here 😉

**Review by naumch1k — APPROVED** [review]
@NazarPauk Congratulations on your approved pull request 👏👏👏

Just a quick reminder: make sure to switch to the `main` branch on your local machine and delete the `docs-media-requests `branch to keep your local version clean. Also, always remember to run the `git pull` command before creating a new feature branch next time. This ensures that your new branch is aligned with the latest changes from GitHub.

Keep excelling!
